packages/react-components/src/components/trace-context-component.tsx

```diff
@@ -435,7 +435,7 @@ export class TraceContextComponent extends React.Component<TraceContextProps, Tr
                     unitController: this.unitController,
                     outputWidth: this.state.style.width,
                     backgroundTheme: this.state.backgroundTheme,
-                    setChartOffset: this.setChartOffset
+                    setChartOffset: this.setChartOffset,
```

Comment:
Were you trying to add experimentName here? Would it be better instead of adding only for specific outputs?